src/uct/api/uct.h

```diff
@@ -2697,6 +2942,74 @@ UCT_INLINE_API unsigned uct_iface_progress(uct_iface_h iface)
 }
 
 
+/**
+ * @ingroup UCT_CLIENT_SERVER
+ * @brief Open a connection manager.
+ *
+ * Open a specific connection manager. All client server connection
```

Comment:
Do we have unspecific connection manager ? the use of `specific` is a bit confusing.

Reply:
@shamisp there is no unspecific connection manager :)